### PR TITLE
Add commonly requested static files

### PR DESCRIPTION
**Pull request by pitpalme**
Current clients request not only favicon.ico, but a few more images,
which are added with this commit.

### CODE REVIEW

File: resources/robots.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+User-Agent: *
+Disallow: /auth
+Disallow: /admin
```
Comment (christophmeissner) [inline resources/robots.txt:3]: `Disallow: /helpdesk`, as well? After all, it's available for authenticated users only anyway.

File: non_logged_in_area/templates/base_non_logged_in.html [hunk 2]
```diff
@@ -11,6 +11,12 @@
 
     <title>Volunteer Planner</title>
 
+    <link rel="icon" type="image/x-icon" href="{% static "img/favicon.ico" %}">
```
Comment (christophmeissner) [inline non_logged_in_area/templates/base_non_logged_in.html:14]: I think those links need to go to `<ROOT>\templates/base.html`, too.

**Review by christophmeissner — CHANGES_REQUESTED** [review]

File: non_logged_in_area/templates/base_non_logged_in.html (hunk 2 above)
Comment (pitpalme) [reply non_logged_in_area/templates/base_non_logged_in.html:14]: Arent't they?

Comment (ChristophMeissner-CGI) [reply non_logged_in_area/templates/base_non_logged_in.html:14]: Yes, they are. I didn't see it, sorry. 🙈

Comment (christophmeissner) [reply non_logged_in_area/templates/base_non_logged_in.html:14]: Yes, they are. I didn't see it, sorry. 🙈

**Review by christophmeissner — APPROVED** [review]

**Review by ChristophMeissner-CGI — APPROVED** [review]

**Pull request merged**